modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -168,11 +168,10 @@ You can disable DHCP on a single interface, such as when there are two or more n
 
 [source,terminal]
 ----
-ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
-ip=::::core0.example.com:enp2s0:none
+bond=bond0:enp1s0,enp2s0:mode=active-backup
+ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:bond0:none ip=enp1s0:none::00:1A:4A:16:01:5A ip=enp2s0:none::00:1A:4A:16:01:5A
 ----
```

Comment:
Oops, this it wrong,  Revert this.  This was not a  bonding config.

We want the original:
```shell
ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
ip=::::core0.example.com:enp2s0:none
```